src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -86,6 +86,12 @@ protected function populateState($ordering = 'id', $direction = 'asc')
 
 		// Get pagination request variables
 		$limit = $app->getUserStateFromRequest('global.list.limit', 'limit', $app->get('list_limit'), 'int');
+		$list = $app->input->get('list');
+		if (isset($list['limit']) && $list['limit'])
+		{
```

Comment:
@Sachintechjoomla Maybe you need to take the reference of context in getUserStateFromRequest
		$limit = $app->getUserStateFromRequest('global.list.limit', 'limit', $app->get('list_limit'), 'int'); => 		$limit = $app->getUserStateFromRequest($this->content . '.list.limit', 'limit', $app->get('list_limit'), 'int');

Reply:
Done